### PR TITLE
Fixed some typos and spelling in the JavaPlugin user guide file

### PR DESCRIPTION
**Pull request by HugooB**
Hello! I read the JavaPlugin user guide and encountered some typos. I did a quick scan of the document for more errors and fixed them in this pull request. 

**Any of the checked boxes below indicate that I took action:**

- [x] Reviewed the [Contribution Guidelines](https://github.com/gradle/gradle/blob/master/.github/CONTRIBUTING.md).
- [x] Signed the [Gradle CLA](http://gradle.org/contributor-license-agreement/).
- [ ] Ensured that basic checks pass: `./gradlew quickCheck`~~

For all non-trivial changes that modify the behavior or public API:

- [ ] Before submitting a pull request, I started a discussion on the [Gradle developer list](https://groups.google.com/forum/#!forum/gradle-dev),
      the [forum](https://discuss.gradle.org/) or can reference a [JIRA issue](https://issues.gradle.org/secure/Dashboard.jspa).
- [ ] I considered writing a design document. A design document can be
brief but explains the use case or problem you are trying to solve,
touches on the planned implementation approach as well as the test cases
that verify the behavior. Optimally, design documents should be submitted
as a separate pull request. [Samples](https://github.com/gradle/gradle/tree/master/design-docs)
can be found in the Gradle GitHub repository. Please let us know if you need help with
creating the design document. We are happy to help!
- [ ] The pull request contains an appropriate level of unit and integration
test coverage to verify the behavior. Before submitting the pull request
I ran a build on my local machine via the command
`./gradlew quickCheck <impacted-subproject>:check`.
- [x] The pull request updates the Gradle documentation like user guide,
DSL reference and Javadocs where applicable.

### CODE REVIEW

**Comment by HugooB** [comment]
Fixed 2 more spelling errors in the user files!

File: subprojects/docs/src/docs/userguide/javaPlugin.xml [hunk 1]
```diff
@@ -1564,7 +1564,7 @@
             <sample id="manifest" dir="userguide/tutorial/manifest" title="Customization of MANIFEST.MF">
                 <sourcefile file="build.gradle" snippet="add-to-manifest"/>
             </sample>
-            <para>You can create stand alone instances of a <classname>Manifest</classname>. You can use that for example,
+            <para>You can create standalone instances of a <classname>Manifest</classname>. You can use that for example,
```
Comment (bmuschko) [inline subprojects/docs/src/docs/userguide/javaPlugin.xml]: The word "stand-alone" should actually contain a dash. http://www.dictionary.com/browse/standalone. Would you mind fix that?

Comment (HugooB) [inline subprojects/docs/src/docs/userguide/javaPlugin.xml]: You're right, my bad. I'll fix it!

**Comment by HugooB** [comment]
@bmuschko Fixed it!

**Comment by bmuschko** [comment]
@HugooB OK, thanks. Before merging can you please rebase your branch. There're some merge conflicts right now.

**Comment by HugooB** [comment]
Rebased it! Should be mergeable now

**Comment by bmuschko** [comment]
Thanks for the pull request. It has been merged!

**Pull request merged**